Source/Parallelization/WarpXRegrid.cpp

```diff
@@ -284,22 +282,22 @@ WarpX::RemakeLevel (int lev, Real /*time*/, const BoxArray& ba, const Distributi
 }
 
 void
-WarpX::ComputeCostsHeuristic (amrex::Vector<std::unique_ptr<amrex::Vector<amrex::Real> > >& costs)
+WarpX::ComputeCostsHeuristic (amrex::Vector<std::unique_ptr<amrex::Vector<amrex::Real> > >& a_costs)
 {
     for (int lev = 0; lev <= finest_level; ++lev)
     {
-        auto & mypc = WarpX::GetInstance().GetPartContainer();
-        auto nSpecies = mypc.nSpecies();
+        auto & mypc_ref = WarpX::GetInstance().GetPartContainer();
+        auto nSpecies = mypc_ref.nSpecies();
```

Comment:
Fixes the warning:
`./Source/Parallelization/WarpXRegrid.cpp:291:16`: warning: declaration of ‘mypc’ shadows a member of ‘WarpX’ [-Wshadow]
         auto & mypc = WarpX::GetInstance().GetPartContainer();
In file included from ./Source/Parallelization/WarpXRegrid.cpp:9:0:
`./Source/WarpX.H:617:45`: note: shadowed declaration is here
     std::unique_ptr<MultiParticleContainer> mypc;

Reply:
maybe rename: `mypc` -> `m_mypc` if its a `WarpX::` member variable.

Reply:
Agreed, this could be in a subsequent PR.